CDTDatastore/HTTP/CDTURLSessionTask.m

```diff
@@ -57,7 +57,7 @@ @interface CDTURLSessionTask ()
 
 @implementation CDTURLSessionTask
 
-- (nullable instancetype)init
+- (instancetype)init
```

Comment:
So this isn't `nullable` because it is always `nil`, rather than possibly being `nil`?